components/dashboard/src/data/prebuilds/prebuild-logs-emitter.ts

```diff
@@ -11,7 +11,13 @@ import { matchPrebuildError, onDownloadPrebuildLogsUrl } from "@gitpod/public-ap
 import { Disposable } from "@gitpod/gitpod-protocol";
 import { ApplicationError, ErrorCodes } from "@gitpod/gitpod-protocol/lib/messaging/error";
 
-export function usePrebuildLogsEmitter(prebuildId: string, taskId: string) {
+/**
+ *
+ * @param prebuildId ID of the prebuild to watch
+ * @param taskId ID of the task to watch. If `null`, watch will not be started.
+ * @returns
+ */
+export function usePrebuildLogsEmitter(prebuildId: string, taskId: string | null) {
```

Comment:
We very rarely use `null`, and almost exclosively `undefined`. Let's stick witht hat convention.